packages/synapse-react-client/test/containers/dataaccess/ReviewerDashboard.integration.test.tsx

```diff
@@ -40,7 +40,24 @@ jest
     return <div data-testid={HISTORY_DASHBOARD_TEST_ID}></div>
   })
 
-function renderComponent() {
+// By default, ensure the user has permission to see all tabs before trying to render them
+function renderComponent(
+  isACTMember: boolean = true,
+  isARReviewer: boolean = true,
+) {
+  server.use(
+    rest.get(
+      `${getEndpoint(BackendDestinationEnum.REPO_ENDPOINT)}${USER_BUNDLE}`,
+      async (req, res, ctx) => {
+        const response: UserBundle = {
+          userId: MOCK_USER_ID.toString(),
+          isACTMember: isACTMember,
+          isARReviewer: isARReviewer,
+        }
+        return res(ctx.status(200), ctx.json(response))
+      },
+    ),
+  )
```

Comment:
Move mocked response into renderComponent and add `isACTMember` and `isARReviewer` as props to consolidate code a bit